src/traces/scattergeo/hover.js

```diff
@@ -0,0 +1,105 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Fx = require('../../plots/cartesian/graph_interact');
+var Axes = require('../../plots/cartesian/axes');
+
+var getTraceColor = require('../scatter/get_trace_color');
+var attributes = require('./attributes');
+
+
+module.exports = function hoverPoints(pointData) {
+    var cd = pointData.cd,
+        trace = cd[0].trace,
+        xa = pointData.xa,
+        ya = pointData.ya,
+        geo = pointData.subplot;
+
+    if(cd[0].placeholder) return;
```

Comment:
this is a little mysterious, at least as someone who doesn't look at Plotly.js code daily. Maybe a comment here?